src/icons/Status.js

```diff
@@ -132,9 +90,7 @@ export function Loading({ className }) {
     return (
         <svg
             xmlns="http://www.w3.org/2000/svg"
-            width="48"
-            height="48"
-            viewBox="22 22 44 44"
+            viewBox="0 0 48 48"
```

Comment:
Loader status is also broken:

![broken-loader](https://user-images.githubusercontent.com/185449/63798248-06a76380-c90a-11e9-9022-6677af2d10e7.gif)

Reply:
Oops this one should have had a custom viewBox.